website/addons/owncloud/README.md

```diff
@@ -0,0 +1,18 @@
+OSF ownCloud Addon
+
+
+## Setup:
+1. Copy settings/local-dist.py to settings/local.py and change the necessary settings.
+2. Make sure to list your server under `trusted_servers`
```

Comment:
Is this necessary/true? It looks like a user can specify a custom host when connecting (which makes sense, given ownCloud's model)